crates/sui-indexer-alt-e2e-tests/src/lib.rs

```diff
@@ -594,6 +598,19 @@ impl OffchainCluster {
         Ok(sequence_number as u64)
     }
 
+    pub async fn write_checkpoint(&self, checkpoint_data: CheckpointData) -> anyhow::Result<()> {
```

Comment:
(this may need to be moved depending on how the comments related to `ingestion_temp_dir` and `ClientArgs` pan out).